Improve readme of repository (#3647)
FAForever/fa: loc/guidelines.md

Translation guidelines
----------------------

1) *Compliance with the game's UI*

> - Text should never overflow from anywhere

Reviewer: ```suggestion
- Text should never overflow
```

> - As much as possible, try to keep a few pixels of margin between the text and its parent element boundaries

Reviewer: ```suggestion
- Maintain a few pixels of margin between the text and its parent element boundaries
```
